src/components/explore/TranscriptContentPage.tsx

```diff
@@ -55,6 +56,9 @@ const TranscriptContentPage: FC<ITranscriptContentPage> = ({
       isActive: idx === pathnameArray.length - 1,
     };
   });
+
+  const allRoutes = routes.filter((route) => !LanguageCodes.includes(route.name))
```

Comment:
can we have a comment here too, why we filter?
e.g `remove language codes from paths passed to breadcrumbs`